api/views/views.py

```diff
@@ -617,6 +618,7 @@ def get_queryset(self):
             "favorite_min_rating",
             "image_scale",
             "save_metadata_to_disk",
+            "default_timezone",
```

Comment:
Do we need to add datetime_rules here?

Reply:
Yes, good catch! When the value is missing, it will cause a second query to fetch the value.